pkg/controller/appsync/helpers.go

```diff
@@ -55,6 +55,7 @@ func (c *Controller) syncApps(ctx context.Context, apps *appsync.AppsResponse) *
 		realm.AgencyBackgroundColor = strings.ToLower(app.AgencyColor)
 		realm.AgencyImage = app.AgencyImage
 		realm.DefaultLocale = app.DefaultLocale
+		realm.UserReportLearnMoreURL = app.WebReportLearnMoreURL
```

Comment:
Will this field always be present? Should we have a fallback?

Reply:
if it's empty, the learn more link is omitted
https://github.com/google/exposure-notifications-verification-server/blob/cc67c9c3a2f8780995b268610a5caf0bd368407e/assets/enx-redirect/report/index.html#L61